src/views/DataSessionsView.vue

```diff
@@ -58,7 +54,7 @@ function tabColor(index) {
 }
 
 function tabActive(index) {
-  if (dataSessions.value[index] && tab.value === dataSessions.value[index].id) {
+  if (dataSessions.value[index] && userDataStore.activeSessionId === dataSessions.value[index].id) {
```

Comment:
also here though I suspect it has to do with which tab is active but still not sure what it's doing in the UI